### DISCUSSION TITLE
BUILD_MONAI inside requirements.txt

### DISCUSSION THREAD

**Discussion by masadcv** [Project-MONAI/MONAI#2732] [General]
We have a MONAILabel app that depends on MONAI's CRF module. This module requires MONAI to be built using:

`BUILD_MONAI=1 pip install git+https://github.com/Project-MONAI/MONAI#egg=monai`

As mentioned [here](https://github.com/Project-MONAI/MONAILabel/issues/262), we need to include this monai installation in the requirements.txt for the relevant app. 

My question is, is there a neat way of including monai in requirements.txt while also compiling to get the relevant CRF binaries?

Many thanks for your help!

Muhammad Asad

cc: @charliebudd

**Comment by Nic-Ma** [comment]
Hi @masadcv ,

Thanks for raising this topic.
I am afraid we don't support this feature so far, @wyli @madil90 @IsaacYangSLA please feel free to correct me if I am wrong.

Thanks.

**Reply by masadcv** [reply]
Thanks!
